Pipfile

```diff
@@ -14,6 +14,8 @@ google-cloud-pubsub = "*"
 requests-toolbelt = "*"
 tenacity = "*"
 google-cloud-storage = "*"
+splinter = "*"
+selenium = "*"
```

Comment:
Doesn't look like `selenium` is used, can be removed.

Reply:
Added as driver to splinter now and not a depenency